package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "unity-request-manual-activation-file",
-  "version": "2.0.0",
+  "version": "2.2.0",
```

Comment:
Should we bump the major instead since this is breaking change?

Reply:
After thinking about it, bumping minor instead of major seems to be the right solution because users are most likely using the following:

```yaml
- name: Unity - Request Activation File
  uses: game-ci/unity-request-activation-file@v2
```

And we actually want the actions to fail and print link to activation docs. I think it's fine, I'm resolving this conversation 👍